cmd/git/branch.go

```diff
@@ -0,0 +1,78 @@
+package git
+
+import (
+  "fmt"
+  "io/ioutil"
+  "log"
+  "strings"
+
+  "github.com/Originate/gt/cmd/util"
+)
+
+
+func EnsureDoesNotHaveBranch(branchName string) {
+  if HasBranch(branchName) {
+    util.ExitWithErrorMessage(fmt.Sprintf("A branch named '%s' already exists", branchName))
```

Comment:
Thats not descriptive enough for my taste. I'd be good with something like `util.ExitWithUserError`